markup.go

```diff
@@ -113,6 +114,14 @@ func (l *EventListener) PreventDefault() *EventListener {
 	return l
 }
 
+// StopPropagation prevents further propagation of the current event in the capturing and bubbling phases.
```

Comment:
Just curious, is that your personal preference, or do you think that it's best practice (listed somewhere else)?

I agree for this comment, because it's top level, that it's nicer to wrap it to 80-100~ish lines (which is optimal for reading).

But there are other comment types that provide very specific detail, like inline notes, that I don't mind keeping very long and single-line. That way they're out of sight except when you really need to read about that particular detail. Here's an [example](https://github.com/shurcooL/markdownfmt/blob/b0700cd85e465d93db670e04763fc6f4070e6e76/markdown/main.go#L379) of what I mean. Disclaimer: I use a computer where I can scroll horizontally as easily as I can scroll vertically.